src/main/java/frc/robot/Constants.java

```diff
@@ -43,32 +55,71 @@ public static class Elevator {
 
         public static final FFConfig FF_CONFIG = new FFConfig(0.26);
 
-        public static final FollowerConfig LEFT_MOTOR = new FollowerConfig(RIGHT_MOTOR, false, 23);
+        public static final FollowerConfig LEFT_MOTOR = new FollowerConfig(RIGHT_MOTOR, false, 25);
         public static final double upperSetpoint = 1.0;
         public static final double lowerSetpoint = 0.0;
 
         public static final double MOTOR_SPEED = 0.5;
         public static final double MAX_ERROR = 0.1;
     }
 
-    public static final class Manipulator {
+    public static final class Manipualtor {
         public static final MotorConfig MOTOR = new MotorConfig(
             24, 
             40, 
             false, 
             Mode.COAST
         );    
 
-        public static final BeamBreakConfig BEAM_BREAK = new BeamBreakConfig(1);
+        public static final BeamBreakConfig BEAM_BREAK = new BeamBreakConfig(6);
+
+        public static final double SPEED = 0.9;
+    }
+
+    public static final class Indexer {
+        public static final MotorConfig MOTOR = new MotorConfig(
+            23, 
+            40, 
+            true, 
+            Mode.COAST
+        );    
 
-        public static final double SPEED = 0.8;
+        public static final double SPEED = 0.5;
     }
 
+    public static final class Intake {
+        // 20 for algea picjup, 35 for algea drop off
+        public static final MotorConfig PIVOT = new MotorConfig(
+            22, 
+            40, 
+            true, 
+            new PIDConfig(0.08),
+            Mode.COAST
+        );    
+
+        public static final MotorConfig SPIN = new MotorConfig(
+            21, 
+            40, 
+            false, 
+            Mode.COAST
+        );    
+
+        public static final double GEAR_RATIO = 25.0 * 2.1;
+        public static final boolean TUNING_MODE = true;
+        public static final FFConfig ff = new FFConfig(0.5, 0.0, 0.0);
+        public static final AbsoluteEncoderConfig ENCODER = new AbsoluteEncoderConfig(2, new Rotation2d(360), false);
+        public static final double UP_SETPOINT = 0.0;
```

Comment:
again enum add algae pickup and algead drop of along with idle, and coral pickup